test/extended/builds/start_build.go

```diff
@@ -52,6 +52,32 @@ var _ = g.Describe("builds: parallel: oc start-build", func() {
 		})
 	})
 
+	g.Describe("override environment", func() {
+		g.It("should accept -e FOO=bar", func() {
+			g.By("starting the build with -e FOO=bar")
+			out, err := oc.Run("start-build").Args("sample-build", "--follow", "--wait", "-e", "FOO=bar").Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+			g.By(fmt.Sprintf("verifying the build output contains the env var"))
+			o.Expect(out).To(o.ContainSubstring("FOO=bar"))
+			g.By(fmt.Sprintf("verifying the build %q status", out))
+			build, err := oc.REST().Builds(oc.Namespace()).Get("sample-build-1")
+			o.Expect(err).NotTo(o.HaveOccurred())
+			o.Expect(build.Status.Phase).Should(o.BeEquivalentTo(buildapi.BuildPhaseComplete))
+		})
+
+		g.It("should allow to change build log level", func() {
+			g.By("starting the build with --build-loglevel=5")
+			out, err := oc.Run("start-build").Args("sample-build", "--follow", "--wait", "--build-loglevel=1").Output()
```

Comment:
@bparees also I updated the other test and added one extra variable that I check to not change. I think this will be testing exactly same thing as setting this parameter is equivalent to -e BUILD_LOGLEVEL=X